package.json

```diff
@@ -5,13 +5,35 @@
   "name": "@microsoft/msgraph-sdk-core",
   "version": "1.0.0-preview.3",
   "description": "Core functionalities for the Microsoft Graph JavaScript SDK",
-  "main": "index.js",
-  "types": "index.d.ts",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "module": "./dist/index.js",
+  "exports": {
+    ".": "./dist/index.js"
+  },
+  "files": [
+    "dist",
+    "src"
+  ],
   "scripts": {
-    "build": "tsc -p tsconfig.json",
-    "test": "mocha tests/**/*.js",
-    "lint": "eslint . --ext .ts",
-    "lint:fix": "eslint . --ext .ts --fix"
+    "prepare": "husky",
+    "build": "npm run updateVersion && npm run prettier:check && npm run clean && npm run build:compile",
```

Comment:
@baywet the build step runs the script to update the version first before "building" commands.